docusaurus/docs/guides/deployments/10-linux/20-router/10-deploy.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Router Deployment
+---
+
+import CliLogin from '/docs/_cli-login.md'
+import LinuxPackageRepo from '/docs/_linux-package-repo.mdx';
+import CliLogFormats from '/docs/_cli-log-levels-and-formats.md'
+
+This article is about deploying a Ziti router as a Linux system service. [This page](/learn/introduction/03-components.md#openziti-router) provides an overview of the architecture.
+
+Routers provide Ziti network entry and exit points for clients and smart routing through the mesh. You must have at least one router that is advertising an edge connection to clients. If you have more than one router, you must configure transport links between them to form the fabric mesh.
+
+## Installation
+
+The router package `openziti-router` provides a systemd service unit and optional bootstrapping. The package depends on the `openziti` package which provides the `ziti` CLI. The easiest way to install both is the cross-platform install script for RPM and Debian distributions.
+
+```text
+curl -sS https://get.openziti.io/install.bash \
+| sudo bash -s openziti-router
+```
+
+<LinuxPackageRepo />
+
+## Bootstrapping
+
+You can opt-in to generating a configuration file and enrolling the router automatically. You will be prompted for the most relevant configuration values when installing interactively unless the answer is already known.
```

Comment:
I feel the same way about this as the controller. I feel like the service should _not_ start unless all answers are provided in some answerfile or somesuch. It's such a common problem to have the advertised addresses wrong that it should be a necessary step.

Routers are also complex to configure. One needs to understand:
* do you want this router to advertise links
* do you want this router to advertise edge

Then there's the whole: do you want this router part of a router group, what interface, what ip address....  it's just too complex imo to try to do in one big automagical bootstrapping. I'd much rather see a "here's where your configuration is, go edit it when/if you need to" and really the only thing that they need to get right is the advertised addresses because that ends up in the server certificate...

Reply:
I found the router will start without the ctrl address and raised this issue to fix it: https://github.com/openziti/ziti/issues/2005

Reply:
I agree. There's no benefit to wrapping every possible config property with this package's bootstrapper or something like `ziti_router_auto_enroll`. Rather, we should document how to configure the router's YAML file for each scenario, whether that config was written from scratch or generated by the bootstrapper as a starting point.

I'll explore in that linked GH issue which answers should be required.